package.json

```diff
@@ -10,7 +10,7 @@
     "build-storybook": "build-storybook",
     "lint": "yarn eslint && stylelint src",
     "eslint": "eslint src stories .storybook",
-    "test:ci": "yarn test && yarn lint",
+    "test:ci": "yarn test --runInBand && yarn lint",
```

Comment:
I've been seeing a number of inexplicable timeouts on CI lately, so trying this (https://jestjs.io/docs/cli#--runinband) as is often recommended for low-resource CI runs.  We'll see whether it ends up helping, but it shouldn't hurt.